configs/config-ccm-psk-tls1_2.h

```diff
@@ -83,6 +83,6 @@
  */
 #define MBEDTLS_SSL_MAX_CONTENT_LEN             512
 
-#include "mbedtls/check_config.h"
+#include "check_config.h"
```

Comment:
The README.txt file in the configs directory states that there are two options to use these config files
>1. Replace the default file include/mbedtls/config.h with the chosen one.
   (Depending on your compiler, you may need to adjust the line with
   #include "mbedtls/check_config.h" then.)
>
>2. Define MBEDTLS_CONFIG_FILE and adjust the include path accordingly.

So in case (1) the user has to do nothing and just copy the file over as stated in the readme (actually, I think the information in brackets is no longer relevant). However, in the case of (2) the user will probably need to add `./configs` to the include path as well as `./include/mbedtls` (because this patch removes the namespacing bit). Also, note that without this change, case (1) will also work with the `mbedtls/` prefix, its just not ideal and its already properly documented.

So I am not too sure that we should remove the namespacing from these config files. But if we do want to remove, we should at least fix the documentation accordingly.

Reply:
I'll revert the changes in the configs